src/styles/book.css

```diff
@@ -0,0 +1,219 @@
+body {
+  background-color: #efefef;
+}
+
+#main-container {
+  display: flex;
+  flex-direction: column;
+  padding: 15px 77px;
+}
+
+.display-books {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  border: 1px solid #e8e8e8;
+  padding: 20px;
+  margin-bottom: 20px;
+  background-color: white;
+}
+
+#nav-container {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  border-bottom: 1px solid #e8e8e8;
+  background-color: white;
+}
+
+#nav-container h1 {
+  color: #0290ff;
+  font-size: 1.875rem;
+  margin-left: 5rem;
+}
+
+#pages-container {
+  display: flex;
+  align-items: center;
+}
+
+#pages-container li {
+  list-style-type: none;
+  padding-left: 20px;
+  padding-right: 20px;
+  letter-spacing: 1px;
+}
+
+.user-icon {
+  border: 1px solid black;
+  padding: 13px;
+  border-radius: 24rem;
+  color: #0290ff;
```

Comment:
-[ ] Kindly consider updating the border color of the account Icon to `#e8e8e8` which is given in the Design System 👏🏼 

### Your Design
![Screenshot from 2023-05-20 13-59-40](https://github.com/Shaheryar0054/bookstore-using-react/assets/68030297/4dc355e1-adb1-4fa1-803f-9254df8f6367)

### System Design
![Screenshot from 2023-05-20 14-00-11](https://github.com/Shaheryar0054/bookstore-using-react/assets/68030297/80888831-33d1-4a1e-8988-e595f4819efd)